cime_config/acme/machines/template.case.test

```diff
@@ -66,8 +66,8 @@ def _main_func(description):
         sys.exit(1 if test_results.failed > 0 else 0)
 
     caseroot, testname = parse_command_line(sys.argv, description)
-    case = Case(caseroot)
-    success = case_test(case, testname)
+    with Case(caseroot) as Case:
```

Comment:
Case should be case here.   Have you run the scripts_regression_tests on this PR?